vite.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
```

Comment:
I think migrating to Vite is the right direction but I'd make a separate PR for it and also should remove webpack.

Reply:
Removed, will make a separate PR.
I tried removing webpack but found that karma config is still dependent on it. Maybe change the test script so that build is run before testing?